sql/executor.go

```diff
@@ -457,6 +457,7 @@ func (e *Executor) execStmt(stmt parser.Statement, planMaker *planner) (Result,
 		} else if planMaker.txn.Proto.Status == roachpb.ABORTED {
 			// Reset to allow starting a new transaction.
 			planMaker.resetTxn()
+			result.PGTag = (*parser.RollbackTransaction)(nil).StatementTag()
```

Comment:
(commenting on github because reviewable doesn't understand that the branch can move backwards)

The fact that postgres sends `ROLLBACK` as the command tag for failed `COMMIT` definitely needs a comment so we don't try to make the same change again.